app-text/chm2pdf/chm2pdf-0.9.2-r1.ebuild

```
@@ -0,0 +1,42 @@
+# Copyright 1999-2020 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{6,7,8,9} )
+
+inherit python-single-r1
+
+DESCRIPTION="A script that converts a CHM file into a single PDF file"
+HOMEPAGE="https://code.google.com/p/chm2pdf/"
+SRC_URI="https://github.com/Zaryob/chm2pdf/releases/download/0.9.2/chm2pdf-0.9.2.tar.gz"
+
+# EGIT_REPO_URI="https://github.com/Zaryob/chm2pdf"
+# inherit git-r3
+
+LICENSE="GPL-2"
+KEYWORDS="amd64 x86"
```

Drop keywords to unstable when bumping a package, and since this is a new package again you'll need to drop keywords for architectures you haven't tested on, so probably:

```suggestion
KEYWORDS="~amd64"
```